app/livechat/server/business-hour/index.ts

```diff
@@ -0,0 +1,5 @@
+import { BusinessHourManager } from './BusinessHourManager';
+import { SingleBusinessHour } from './Single';
+import { cronJobs } from '../../../utils/server/lib/cron/Cronjobs';
+
+export const businessHourManager = new BusinessHourManager(new SingleBusinessHour(), cronJobs);
```

Comment:
I'm not sure we should hard code the `new SingleBusinessHour()` when creating the `BusinessHourManager`.
Maybe we'll need to change it in order to support the `EE` which will allow selecting the business hour configuration: `SINGLE` or `MULTIPLE`. A callback would work fine here in order to get the default `SingleBusinessHour` class to instantiate the `BusinessHourManager`.

Please, think about it.